### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by zhanggbj**
Hi,

This patch included two major change. Thanks.
1. Combined bmp project and made separated directory for different project.
2. Updated ci.

### CODE REVIEW

File: ci/README.md [hunk 1]
```diff
@@ -1,232 +0,0 @@
-
```
Comment (maximilien) [inline ci/README.md]: Please move this file as `docs/deploying_concourse_on_sl_user_guide.md`

File: bin/build [hunk 2]
```diff
@@ -7,4 +7,5 @@ echo -e "\n Formatting packages..."
   go fmt ./...
 
 echo -e "\nGenerating Binary..."
-go build -o $(dirname $0)/../out/sl_stemcells ./main/main.go
+go build -o $(dirname $0)/../out/sl_stemcells ./main/stemcells/main.go
```
Comment (maximilien) [inline bin/build]: I like the simplication of `sl_stemcells` to `stemcells`. Let's change the name.

/cc @jianqiu will need to update pipeline YMLs with this change

File: ci/pipelines/Bosh-Stemcell-Dockerfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+FROM karafelix/slci
```
Comment (maximilien) [inline ci/pipelines/Bosh-Stemcell-Dockerfile]: same deal: `bosh-stemcell-dockerfile`

File: ci/pipelines/Dockerfile [hunk 4]
```diff
@@ -12,9 +12,9 @@ RUN apt-get update && \
 # BEGIN stemcell requirements
```
Comment (maximilien) [inline ci/pipelines/Dockerfile]: `dockerfile`

**Comment by maximilien** [comment]
@zhanggbj I am merging but let's address my comments after. 

Thanks. Best :+1:

**Comment by zhanggbj** [comment]
@maximilien Thanks for your review. I'll update these in next patch.

**Pull request merged**